aws-blog-event-driven-batch-analytics/src/main/java/com/amazonaws/bigdatablog/edba/emr/ProcessVendorTrasactions.java

```diff
@@ -0,0 +1,87 @@
+package com.amazonaws.bigdatablog.edba.emr;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+
+import org.apache.spark.SparkConf;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.api.java.function.Function;
+import org.apache.spark.sql.DataFrame;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.RowFactory;
+import org.apache.spark.sql.SQLContext;
+import org.apache.spark.sql.SaveMode;
+import org.apache.spark.sql.types.DataTypes;
+import org.apache.spark.sql.types.StructField;
+import org.apache.spark.sql.types.StructType;
+
+import com.amazonaws.auth.AWSSessionCredentials;
+import com.amazonaws.auth.InstanceProfileCredentialsProvider;
+import com.google.common.collect.ImmutableMap;
+
+import scala.collection.Seq;
+
+public class ProcessVendorTrasactions {
+
+	final static String redshiftJDBCURL="jdbc:redshift://<<YourClusterEndPoint>>:5439/test?user=<<YourUser>>&password=<<YourPassword>>";
```

Comment:
This cluster endpoint, user, etc, should be in a separate properties file for the Redshift activities.